mariadb.spec

```diff
@@ -1307,6 +1349,52 @@ fi
 %endif
 
 %changelog
+* Tue Jan 10 2017 Michael Bayer <mbayer@redhat.com> - 3:10.1.20-1
+- Backport Fedora changes since Nov 16
+- Update to version 10.1.20
+
+* Tue Jan 03 2017 Honza Horak <hhorak@redhat.com> - 3:10.1.20-3
```

Comment:
I would not merge Fedora changelogs, just summarize important changes for the end-users in your single changelog for 10.1.20. I guess only one relevant is just - Update to version 10.1.20
even "Backport Fedora changes since..." is not interesting to the end-user IMHO.